config/initializers/rack/attack.rb

```diff
@@ -22,7 +22,7 @@ class Rack::Attack
     end
   end
 
-  throttle("site_hits", limit: 100, period: 2) do |request|
+  throttle("site_hits", limit: 20, period: 2) do |request|
```

Comment:
In going to the site I feel like it's a legit situation where a user could visit the home page and then click into an article within a second, and you'd pass 20 hits this way. So maybe cautiously bump this to 40 in two seconds, which is an improvement from 100 but still has a margin of safety for legit traffic.

Reply:
Can do!